packages/palette-docs/content/docs/components/TimeRemaining.mdx

```diff
@@ -4,16 +4,16 @@ name: TimeRemaining
 
 <Playground>
   <TimeRemaining
-    countdownEnd="2019-01-03T20:15:00.000-04:00"
-    currentTime="2019-01-01T12:00:30.000-04:00"
+    currentTime="2019-01-02T20:15:00.000-04:00"
+    endDate="2019-01-03T20:15:00.000-04:00"
     highlight="purple100"
   />
 </Playground>
 
 <Playground title="Out of time">
   <TimeRemaining
-    countdownEnd="2019-01-03T20:15:00.000-04:00"
-    currentTime="2019-01-05T12:00:00.000-04:00"
+    currentTime="2019-01-02T20:15:00.000-04:00"
+    endDate="2019-01-03T20:15:00.000-04:00"
```

Comment:
I think a previous comment got eaten by rebase, but why change `countdownEnd`? I'm cool with the change if certain there aren't a lot of consumers that would need to be updated, but maybe change to `endTime` to be in alignment with `currentTime`, or even `endDateTime` (and `currentDateTime`) -- either way.

Reply:
Reverted this.